ProcessMaker/processmaker
# Add Security Log API Documentation

## Issue & Reproduction Steps
TICKET: [FOUR-5978](https://processmaker.atlassian.net/browse/FOUR-5978)

As a process architect, I want to have documentation of the Security Logs API, so I can follow up on security logs

## Solution
- Add API documentation to Swagger

## How to Test

1. Navigate to `/api/documentation` in your local instance. 
2. Authorize yourself via Bearer Token
3. Locate the 'Security Logs' location
4. 'Try Out' both APIs
5. Ensure all fields work properly
6. Ensure all properties are listed in the example schema

## Related Tickets & Packages
NONE

## Code Review Checklist
